README.md

```diff
@@ -92,7 +92,8 @@ You can run a single integration test this way: `docker-compose exec web bash -c
 
 By default, when a test fails, it's retried up to 3 times on CI and not retried locally. You can control it by using the environment variable `TEST_RETRY_COUNT`. For example, for five attempts: `docker-compose exec web bash -c "cd test && TEST_RETRY_COUNT=5 rspec --example KEYWORD spec/integration_spec.rb"`.
 
-By default, only unit tests will run for branches on Travis other than `develop` or `master`. In order to run all the tests in any branch it's just necessary to include `[full ci]` in your commit message, and the commit doesn't even need to contain anything, for example: `git commit --allow-empty -m '[full ci] Run all integration tests for this branch'`.
+By default, only unit tests will run for branches on Travis other than `develop` or `master`. In order to run all the tests in any branch it's just necessary to include `[full ci]` in your commit message, and the commit doesn't even need to contain anything, for example: `git commit --allow-empty -m '[full ci] Run all integration tests for this branch'`. Furthermore, if you wish to run only the smoke tests, include `[smoke tests]` in your commit message. Similarly, you can add `[similarity tests]` in your commit message to run only the similarity tests.
```

Comment:
@danielevalverde is the best! This will greatly increase the speed that I can break integration tests! (and then struggle to fix them)